client/js/components/newTrip.js

```diff
@@ -1,90 +1,184 @@
+			<div class="div-input-field date-field">
+				<i class="fa-solid fa-plane-departure icon"></i>
+				<input name="date_start" class="date-input" type="text" placeholder="Start Date">
+			</div>
 
-	for (let input of dateInputs) {
-		input.addEventListener("focus", (event) => {
-			event.target.type = "date";
-			event.target.showPicker();
-		});
-		input.addEventListener("blur", (event) => {
-			event.target.type = "text";
-		});
-	}
+			<div class="div-input-field date-field">
+				<i class="fa-solid fa-plane-arrival icon"></i>
+				<input name="date_end" class="date-input" type="text" placeholder="End Date">
+			</div>
 
-	//code to post the form data
-	const newTripForm = document.querySelector("#newtrip-form");
-
-	newTripForm.addEventListener("submit", (event) => {
-		event.preventDefault();
-		const formData = new FormData(newTripForm);
-		const data = {
-			holiday_name: formData.get("holiday_name"),
-			date_start: formData.get("date_start"),
-			date_end: formData.get("date_end"),
-			location_name: formData.get("location_name"),
-			img_link: formData.get("img_link"),
-		};
-
-		for (let value in data) {
-			if (!value) {
-				return alert("Please fill out entire form");
-			}
-		}
+			<div id="newtrip-base">
+				<button id="create" type="submit">Create</button>
+			</div>
 
-		if (data.date_start >= data.date_end) {
-			return alert("Ensure that your dates are correct");
-		}
+		</form>
+		</section>
+		`;
+
+		//makes the UI of the start/end dates more responsive
+		const dateInputs = document.querySelectorAll(".date-input");
 
-		axios
-			.post("/api/holidays", data)
-			.then(() => {
-				location.href = "/";
-			})
-			.catch((err) => {
-				return alert(err);
-				//errorBar(err.response.data.message, "error");
+		for (let input of dateInputs) {
+			input.addEventListener("focus", (event) => {
+				event.target.type = "date";
+				event.target.showPicker();
+			});
+			input.addEventListener("blur", (event) => {
+				event.target.type = "text";
 			});
-	});
+		}
+
+		//code to post the form data
+		const newTripForm = document.querySelector("#newtrip-form");
+
+		newTripForm.addEventListener("submit", (event) => {
+			event.preventDefault();
+			const formData = new FormData(newTripForm);
+			const data = {
+				holiday_id: formData.get("holiday_id"),
+				part_name: formData.get("part_name"),
+				date_start: formData.get("date_start"),
+				date_end: formData.get("date_end"),
+				location_name: formData.get("location_name"),
+				img_link: formData.get("img_link"),
+			};
+
+			for (let value in data) {
+				if (!value) {
+					return alert("Please fill out entire form");
+				}
+			}
+
+			if (data.date_start >= data.date_end) {
+				return alert("Ensure that your dates are correct");
+			}
+
+			axios
+				.post("/api/holidayparts", data)
+				.then(() => {
+					location.href = "/";
+				})
+				.catch((err) => {
+					return alert(err);
+					//errorBar(err.response.data.message, "error");
+				});
+		});
+	}
```

Comment:
pretty sure this is where the if-else ends but maybs just put an error bar thing that pops up when both options are fucked like `errorBar("Something went wrong. Try again later", "error")`